helm/observability-bundle/values.yaml

```diff
@@ -68,20 +48,6 @@ apps:
               serviceMonitor:
                 https: true
```

Comment:
Why keep the port and insecureSkipVerify? Also can you add them to proemtheus operator and link it here :D